### DISCUSSION TITLE
Where is the server source code itself?

### DISCUSSION THREAD

**Discussion by MarianoFacundoArch** [abrahamjuliot/creepjs#263] [General]
Hi,
Where is the server source code itself that this API interacts with?
Thanks.

**Comment by abrahamjuliot** [comment]
It's private.

**Comment by MarianoFacundoArch** [comment]
Is there any reason for keeping it private?
What do we don't want to leak to the outside world?

**Reply by abrahamjuliot** [reply]
The main reason is we don't want this used as a fingerprinting library. The goal is to promote research and education. If sharing the server-side code catches interest, it can be discussed here. But, it's not likely to be presented beyond the process and brief snippets.
